mautic/mautic
# [UI] Contact statistics

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | 🔴 
| New feature/enhancement? (use the a.x branch)      | 🟢
| Deprecations?                          | 🔴
| BC breaks? (use the c.x branch)        | 🔴
| Automated tests included?              | 🔴 
| Related user documentation PR URL      | mautic/user-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation-new#... 
| Issue(s) addressed                     | Fixes #... 



## Description

this pr improves the lead stats page by reorganizing the file structure and creating a new modern style. the current layout is a bit outdated. 

by separating the code into more logical components and applying a fresh design, the page will be more user-friendly and visually appealing. 

Dependency:
- #13848 




| Before                                 | After
| -------------------------------------- | ---
|  ![image](https://github.com/mautic/mautic/assets/116097999/e31d824d-26b9-424c-9137-c0a5151ad4ed)                                      | ![image](https://github.com/mautic/mautic/assets/116097999/f3a31a12-1337-4c07-91f0-67f65aead055)




---
### 📋 Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Open Contacts 
3. Create new 
4. Fill anything 
5. Save & Close
6. Open Details and the tab Statistics
7. See the new style
